config/templates/hive-csv-template.yaml

```diff
@@ -4,56 +4,108 @@ metadata:
   name: hive-operator-0.0.1
   namespace: placeholder
   annotations:
-    categories: A list of comma separated categories that your operator falls under.
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
     certified: "false"
     description: OpenShift cluster provisioning and management at scale.
-    containerImage: quay.io/dgoodwin/hive:latest
+    containerImage: quay.io/openshift-hive/hive:v0.0.1
     createdAt: "2019-02-25T14:29:00Z"
-    support: Devan Goodwin
+    repository: https://github.com/openshift/hive
+    support: Hive Team
+    alm-examples: |-
+      [{"apiVersion":"hive.openshift.io/v1","kind":"HiveConfig","metadata":{"name":"hive"},"spec":{"managedDomains":[{"aws":{"credentialsSecretRef":{"name":"my-route53-creds"}},"domains":["my-base-domain.example.com"]}]}}]
 spec:
-  displayName: Hive
-  description: OpenShift cluster provisioning and management at scale.
+  displayName: Hive for Red Hat OpenShift
+  description: |-
+    Hive for Red Hat OpenShift is an operator that runs on top of Kubernetes/OpenShift. Hive can be used to provision
+    and perform initial configuration of OpenShift clusters.
+
+    For provisioning OpenShift, Hive uses the [OpenShift installer](https://github.com/openshift/installer).
+
+    ### Supported cloud providers
+    * AWS
+    * Azure
+    * Google Cloud Platform
```

Comment:
I know this is from README and that doesn't have it, but I think OpenStack could go here now.

Reply:
done